wiring.c

```diff
@@ -116,21 +116,24 @@ unsigned long micros() {
   // Restore SREG
   SREG = oldSREG;
 
-#if F_CPU >= 24000000L && F_CPU < 32000000L
```

Comment:
I believe that this (old) line does the trick withoug duplicating code. if F_CPU is actually 32MHz, it will select the `#else`condition at the bottom of the micros() function

Reply:
I think if it is 32 M, it will enter the 20 M branch.

Reply:
You're absolutely right, my bad...